packages/databyss-ui/components/Editor/slate/ContentEditable.js

```diff
@@ -14,7 +14,7 @@ const toSlateJson = (editorState, pageBlocks) => ({
   document: {
     nodes: pageBlocks.map(block => {
       const textBlock =
-        block.type === 'SOURCE'
+        block.type === 'SOURCE' || block.type === 'TOPIC'
```

Comment:
It seems like we're repeating this OR a lot. Let's replace all with a function like `isAtomicInlineType(type)`